website/_posts/2024-01-08-mainnet-announcement.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: "Mainnet announcement"
+date: 2024-01-08 00:00:00 +0000
```

Comment:
This announcement is for today or 2024-01-08?

Reply:
No. We announce it 2 days ago. so let's keep the same date here